pkgs/by-name/im/immich-go/package.nix

```
@@ -31,20 +31,25 @@ buildGoModule rec {
     '';
   };
 
-  vendorHash = "sha256-jED1K2zHv60zxMY4P7Z739uzf7PtlsnvZyStOSLKi4M=";
+  vendorHash = "sha256-tWpWeqKyXT7bZPE3kpBE15iVQB4zZOsaWEKZ6H8Cwug=";
 
   # options used by upstream:
   # https://github.com/simulot/immich-go/blob/0.13.2/.goreleaser.yaml
   ldflags = [
     "-s"
     "-w"
     "-extldflags=-static"
-    "-X main.version=${version}"
+    "-X github.com/simulot/immich-go/app.Version=${version}"
   ];
 
+  preCheck = ''
+    export HOME=$TMPDIR
+    mkdir -p $HOME
+  '';
```

```suggestion
  nativeCheckInputs = [
    writableTmpDirAsHomeHook
  ];
```
and put this below `preBuild`

---

Thanks for the reference! How could I have found this hook by myself?

---

Not sure. It's pretty new afaik and I only stumbled on it by accident.